src/style.css

```diff
@@ -1 +1,119 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('./fonts/Avenir-Book.ttf') format(truetype);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf) format(truetype);
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-bar_big {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+}
+
+.search-bar_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-bar__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-bar__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search-bar__label_big::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar__label_small::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
these 3 lines could've been comprised in line 45 (.search-bar__label::after) as they are common for both